### PR TITLE
adding dropout layers using tf.nn

### PR DESCRIPTION
**Pull request by Abrar-Mustakim**

### CODE REVIEW

**Comment by simonetgordon** [comment]
Hi,

Thanks for the contribution! Could you comment which issue + issue number (like #[number]) this is addressing in the PR description? Also, please take a look at the details of the failing lint check and try to get it passing.

Cheers, 

Simone

**Comment by simonetgordon** [comment]
Hi,

Thanks for the contribution! Could you comment which issue + issue number (like #[number]) this is addressing in the PR description? Also, please take a look at the details of the failing lint check and try to get it passing.

Cheers, 

Simone
